ocs_ci/ocs/resources/pvc.py

```diff
@@ -131,3 +131,28 @@ def create_multiple_pvc(number_of_pvc=1, pvc_data=None):
         pvc_objs.append(pvc_obj)
         log.info(f'Created Persistent Volume Claim {pvc_name}')
     return pvc_objs
+
+
+def get_pv_for_pvc(pvc_obj):
+    """
+    Function to return a PV object for the given PVC
```

Comment:
This exists already - ocs_ci.ocs.resources.pvc.PVC#backed_pv

Reply:
This function is to get pv object

Reply:
got you, thanks